db_test.go

```diff
@@ -8,13 +8,16 @@ import (
 
 	"github.com/stretchr/testify/suite"
 	"github.com/surrealdb/surrealdb.go"
+	gorilla "github.com/surrealdb/surrealdb.go/pkg/gorilla"
+	"github.com/surrealdb/surrealdb.go/pkg/websocket"
 )
 
 // TestDBSuite is a test s for the DB struct
 type SurrealDBTestSuite struct {
 	suite.Suite
-	db      *surrealdb.DB
-	options []surrealdb.Option
+	db                *surrealdb.DB
+	ws                websocket.WebSocket
```

Comment:
If we are testing the map of web socket implementations, then we don't want the web socket itself I think? Also the db would be tied per implementation. So actually the suite is a map of implementation name (string) to db struct that has a websocket tied with it.

Reply:
in the web socket implementations map, we didn't connect yet,
it just created to WebSocket and applied to options
then we connect it gives us the ws interface with that we connect to db itself and save it into struct as db.

I was thinking to remove Ws and just put an index of the map.